CONTRIBUTING.md

```diff
@@ -40,14 +39,193 @@ If you happen to have a fix for a package, please have a look at the
 issue tracker to see if it doesn't concern an existing issue. If that
 is the case mention it in your pull request.
 
-
 Adding new packages
 -------------------
 
 Information about creating new packages and adding them to repository
 is available in
 [opam's manual](https://opam.ocaml.org/doc/Packaging.html).
 
+The following is a tutorial on the steps to be followed to make your
+package available in opam.ocaml.org:
+
+A quick summary of the steps to be followed are listed below:
```

Comment:
In my experience, we should also mention that in the example below we will publish an imaginary package called `hello` and that this is reflected in the name of its `opam` file in the repository. Maybe even adding something like
```
$ ls
bin/ 
src/
dune-project
hello.opam
```